includes/Database_Upgrader.php

```diff
@@ -159,9 +160,11 @@ public function run_upgrades(): void {
 	 */
 	protected function run_upgrade_routine( string $class, string $version, string $current_version ): void {
 		if ( version_compare( $current_version, $version, '<' ) ) {
-			if ( ! method_exists( $this->injector, 'make' ) ) {
-				return;
-			}
+			/**
+			 * Instance of a migration class.
+			 *
+			 * @var Migrate_Base $routine
+			 */
```

Comment:
Nit: should check for `Migration` interface instead

Reply:
Fixed